### PR TITLE
Log main golang runtime env vars

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/kubernetes/issues/108357

```release-note
NONE
```

/kind cleanup
/priority important-longterm
/sig scalability

### CODE REVIEW

**Comment by wojtek-t** [comment]
/hold

Will add to other core components too if we agree that it's what we want.

**Comment by dims** [comment]
looks like this needs a  `./hack/update-gofmt.sh` :)

**Comment by dims** [comment]
cc @MadhavJivrajani

**Comment by wojtek-t** [comment]
@liggitt - I added it to other components too - PTAL

File: cmd/kube-scheduler/app/server.go [hunk 1]
```diff
@@ -140,6 +140,8 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 	// To help debugging, immediately log version
 	klog.InfoS("Starting Kubernetes Scheduler", "version", version.Get())
```
Comment (liggitt) [inline cmd/kube-scheduler/app/server.go:142]: pre-existing, but for consistency, add version info as well?

```
klog.Infof("Version: %+v", version.Get())
```

Comment (wojtek-t) [reply cmd/kube-scheduler/app/server.go:142]: It's above - just in a different format. Should we change this?

Comment (liggitt) [reply cmd/kube-scheduler/app/server.go:142]: oh, missed it because the line didn't start with `version:`. nah, I wouldn't change it here

**Comment by dims** [comment]
```
/home/prow/go/src/k8s.io/kubernetes/cmd/kubelet/app/server.go:396:2: unstructured logging function "Infof" should not be used
```

**Comment by wojtek-t** [comment]
@dims - fixed, PTAL

**Comment by dims** [comment]
now verify: gofmt tripped up 🤣  :)

**Comment by wojtek-t** [comment]
> now verify: gofmt tripped up rofl :)

<lol> Sorry for that... should be fixed now

**Pull request merged**